corehq/apps/users/templates/users/mobile_workers.html

```diff
@@ -56,9 +56,16 @@
             {% endif %}
             {% if can_add_extra_users and can_access_all_locations %}
               <a class="btn btn-default" href="{% url "user_fields_view" domain %}" id="btn-edit_user_fields">
+                <i class="fa fa-edit"></i>
                 {% trans "Edit User Fields" %}
               </a>
             {% endif %}
+            {% if request|toggle_enabled:"BULK_USER_DELETE" %}
```

Comment:
nit: how about giving this option only to project admin to avoid another toggle and to avoid giving this big power with just a toggle add?

Reply:
I don't like adding a new flag either, but project admin isn't usually the right place to add things, different projects will have different ideas of what a project admin should do.